catalog/app/containers/Bucket/Listing.tsx

```diff
@@ -985,16 +986,9 @@ export function Listing({
           )
         },
       },
-      // TODO: uncomment this after implementing custom filter operators
-      // {
-      //   field: 'type',
-      //   headerName: 'Type',
-      //   type: 'string',
-      //   hide: true,
-      //   // TODO: custom filter operators
-      //   // filterOperators: GridFilterOperator[]
-      // },
-      {
+    ]
+    if (items.some(({ size }) => !!size)) {
```

Comment:
```suggestion
    if (items.some(({ size }) => size != null)) {
```
zero size is a valid case i believe